go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 	k8s.io/api v0.25.5
 	k8s.io/apimachinery v0.25.5
-	k8s.io/apiserver v0.25.0
+	k8s.io/apiserver v0.25.5
```

Comment:
Curious on why specifically 0.25.5? Wondering if these k8s deps could be bumped to the latest patch version for 1.25 release instead (0.25.15) which also contains partial fixes for the http2 rapid-reset CVE-2023-44487 (https://github.com/kubernetes/kubernetes/pull/121201)